client/windows/InfoWindows.h

```diff
@@ -12,14 +12,19 @@
 #include "CWindowObject.h"
 #include "../../lib/FunctionList.h"
 
+VCMI_LIB_NAMESPACE_BEGIN
```

Comment:
What are rules for having these macroses?

Reply:
see f0ecb902209634ae89617a613d68ac024936620c